bbb-webhooks.placeholder.sh

```
@@ -1 +1 @@
-git clone --branch v3.2.2 --depth 1 https://github.com/bigbluebutton/bbb-webhooks bbb-webhooks
+git clone --branch v3.3.0 --depth 1 https://github.com/bigbluebutton/bbb-webhooks bbb-webhooks
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)